pkgs/by-name/od/odin/package.nix

```
@@ -29,11 +30,22 @@ stdenv.mkDerivation (finalAttrs: {
     # which can be provided by a pure Nix expression, for example in a shell.
     ./system-raylib.patch
   ];
-  postPatch = ''
-    rm -r vendor/raylib/{linux,macos,macos-arm64,wasm,windows}
 
-    patchShebangs --build build_odin.sh
-  '';
+  postPatch =
+    lib.optionalString (system == "aarch64-darwin") ''
```

```suggestion
    lib.optionalString (stdenv.hostPlatform.system == "aarch64-darwin") ''
```
This is purer and safer

---

Thanks. I wasn't sure what the best way to pull that in was. I've fixed it in this commit: https://github.com/NixOS/nixpkgs/pull/429353/commits/f4339f6824fb6f2cf4f13cd7b7c1516e8d11a717